cfgov/jinja2/v1/_includes/organisms/secondary-navigation.html

```diff
@@ -15,8 +15,9 @@
 <nav class="o-secondary-navigation
             {{ '' if has_children else 'o-secondary-navigation__no-children' }}"
      aria-label="{{ _('Section navigation') }}">
+    {% set sec_nav_settings_label = _("In this section") %}
```

Comment:
If it matters, we had single quotes before and this is double quotes. Not sure what we usually use for these.

Reply:
I don't know that it matters; I copied [the example we have documented](https://cfpb.github.io/consumerfinance.gov/translation/#do-not-include-the-_-call-in-a-larger-jinja2-template-data-structure).